eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/common/impl/SwitchYardProject.java

```diff
@@ -495,14 +495,28 @@ private String readRawVersionString(MavenProject mavenProject) {
     
     private String readIntegrationVersionPropertyString(MavenProject mavenProject) {
         if (mavenProject.getProperties() != null) {
-            return mavenProject.getProperties().getProperty(_integrationVersionPropertyKey);
+            if (mavenProject.getProperties().getProperty(_integrationVersionPropertyKey) != null) {
+                return mavenProject.getProperties().getProperty(_integrationVersionPropertyKey);
+            } else {
+                // try getting version from the dependency management section
+                for (Dependency dependency : mavenProject.getDependencyManagement().getDependencies()) {
+                    if ("org.jboss.integration.fuse".equals(dependency.getGroupId())) {
+                        return dependency.getVersion();
```

Comment:
just to be sure, we have the resolved value of teh version here? or do we have the written version? (${fuse.version} for instance)